### PR TITLE
Add Wishlists' delete action

### LINKED ISSUE #14: [Dependent on Issue #8, #6] Add Rule - Only Signed in user can delete a wishlist
wishlists#destroy

### LINKED ISSUE #28: [Dependent on Issue #24] Deleting wishlist
In the view that has list of all wishlists, it's possible to delete specific list by clicking button "Delete".

### PR DESCRIPTION
**Pull request by student-tomasz**
Closes #28 
Closes #14

### CODE REVIEW

**Comment by nodzy** [comment]
:+1:

**Comment by Kacper3331** [comment]
👍

File: app/views/wishlists/index.html.erb [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <% @wishlists.each do |wishlist| %>
       <li>
         <%= link_to wishlist.name, "#" -%>
+        <%= button_to "Delete", wishlist_path(wishlist.id), method: :delete -%>
```
Comment (gotar) [inline app/views/wishlists/index.html.erb]: you can replace `wishlist_path(wishlist.id)` with `wishlist`

for rails it's enough

**Comment by gotar** [comment]
+1

**Pull request merged**